src/content/docs/kubernetes-pixie/kubernetes-integration/installation/reduce-ingest.mdx

```diff
@@ -1,114 +1,262 @@
 ---
-title: Reduce data ingest
+title: Reduce data ingested
 tags:
-  - Integrations
-  - Kubernetes integration
-  - Understand and use data
-metaDescription: 'New Relic''s Kubernetes integration: How to install and activate the integration, and what data is reported.'
+    - Integrations
+    - Kubernetes integration
+    - Understand and use data
+metaDescription: 'New Relic''s Kubernetes integration: How to reduce the data ingested.'
 redirects:
-  - /docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data
-  - /docs/integrations/kubernetes-integration/understand-use-data/understand-use-data
-  - /docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#services
-freshnessValidatedDate: never
+    - /docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data
+    - /docs/integrations/kubernetes-integration/understand-use-data/understand-use-data
+    - /docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#services
```

Comment:
@nbaenam since you deleted the governance doc, shouldn't you put a redirect in this combined doc?

`/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance`